debuglog-plugin/src/main/kotlin/com/bnorm/debug/log/DebugLogTransformer.kt

```diff
@@ -99,10 +99,15 @@ class DebugLogTransformer(
       if (expression.returnTargetSymbol != function.symbol) return super.visitReturn(expression)
 
       return DeclarationIrBuilder(pluginContext, function.symbol).irBlock {
-        val result = irTemporary(expression.value)
-        +irDebugExit(function, startTime, irGet(result))
-        +expression.apply {
-          value = irGet(result)
+        if (expression.value.type == typeUnit) {
```

Comment:
Without this change, I noticed that returning early in a function that does not have a return type (therefore Unit) would output `"] = " + Unit.INSTANCE` rather than `"]"`